frontend/public/components/resource-pages.ts

```diff
@@ -56,107 +59,110 @@ import {
   StatefulSetModel,
   StorageClassModel,
   SubscriptionModel,
-  PackageManifestModel,
 } from '../models';
 
 export const resourceDetailPages = ImmutableMap<GroupVersionKind | string, () => Promise<React.ComponentType<any>>>()
-  .set(referenceForModel(ClusterServiceClassModel), () => import('./cluster-service-class' /* webpackChunkName: "cluster-service-class" */).then(m => m.ClusterServiceClassDetailsPage))
```

Comment:
I've avoid reordering the list at this time to keep this PR smaller (since we're in feature freeze)